electrotrans-pojo-bl/src/main/java/decavun2/objects/Repair.java

```diff
@@ -0,0 +1,128 @@
+package decavun2.objects;
+
+
+import ua.com.fielden.platform.entity.DynamicEntityKey;
+
+import java.util.Date;
+
+
+import decavun2.personnel.Person;
+
+
+import ua.com.fielden.platform.entity.ActivatableAbstractEntity;
+import ua.com.fielden.platform.entity.annotation.KeyType;
+import ua.com.fielden.platform.entity.annotation.KeyTitle;
+import ua.com.fielden.platform.entity.annotation.CompanionObject;
+import ua.com.fielden.platform.entity.annotation.CompositeKeyMember;
+import ua.com.fielden.platform.entity.annotation.MapEntityTo;
+import ua.com.fielden.platform.entity.annotation.MapTo;
+import ua.com.fielden.platform.entity.annotation.Observable;
+import ua.com.fielden.platform.entity.annotation.Required;
+import ua.com.fielden.platform.entity.annotation.Title;
+import ua.com.fielden.platform.entity.annotation.DescTitle;
+import ua.com.fielden.platform.entity.annotation.DisplayDescription;
+import ua.com.fielden.platform.entity.annotation.IsProperty;
+import ua.com.fielden.platform.entity.annotation.DescRequired;
+import ua.com.fielden.platform.reflection.TitlesDescsGetter;
+import ua.com.fielden.platform.utils.Pair;
+
+/**
+ * Master entity object.
+ * 
+ * @author Developers
+ *
+ */
+@KeyType(DynamicEntityKey.class)
+@KeyTitle("Repair ID")
+@CompanionObject(RepairCo.class)
+@MapEntityTo
+@DescTitle("Description of Repair")
+@DisplayDescription
+@DescRequired
+public class Repair extends ActivatableAbstractEntity<DynamicEntityKey> {
+
+    private static final Pair<String, String> entityTitleAndDesc = TitlesDescsGetter.getEntityTitleAndDesc(Repair.class);
+    public static final String ENTITY_TITLE = entityTitleAndDesc.getKey();
+    public static final String ENTITY_DESC = entityTitleAndDesc.getValue();
+
+    @IsProperty
+    @MapTo
+    @Title(value = "Repair ID", desc = "The ID of a particular repair.")
+    @CompositeKeyMember(1)
+    private String repairID;
```

Comment:
Again max-len validator, you can use `tgp-composite-key-string` is adds it automatically